src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1,3 +1,17 @@
-export const CommentInfo = () => (
-  <>Put the comment here</>
+export const CommentInfo = ({ comment }) => (
+  <>
```

Comment:
There is no need to wrap the component in a fragment (<>...</>) since it is returning a single parent div element